Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -206,6 +218,16 @@ theorem isVonNBounded_singleton (x : E) : IsVonNBounded 𝕜 ({x} : Set E) := fu
   (absorbent_nhds_zero hV).absorbs
 #align bornology.is_vonN_bounded_singleton Bornology.isVonNBounded_singleton
 
+protected theorem IsVonNBounded.vadd [ContinuousAdd E] {s : Set E}
+    (hs : IsVonNBounded 𝕜 s) (x : E) : IsVonNBounded 𝕜 (x +ᵥ s) := by
+  rw [← singleton_vadd]
+  exact IsVonNBounded.add (isVonNBounded_singleton x) hs
```

Comment:
```suggestion
  exact (isVonNBounded_singleton x).add hs
```

Reply:
AFAIR, this fails for a reason I fail to understand.

Reply:
Hmm, mind leaving a todo asking to fix it?